### PR TITLE
Added Makefile and YAPF

### LINKED ISSUE #55: Add a Makefile and YAPF

### PR DESCRIPTION
**Pull request by takeontom**
Tested like this:

```shell
$ cd Projects
$ cookiecutter ~/Projects/django-template/
$ cd whatever
$ mkvirtualenv whatever
$ make reset
$ tox
$ ./manage.py runserver
```

Fixed files (mainly the settings files) which were failing the yapf lint.

Fixes #55

### CODE REVIEW

File: {{cookiecutter.project_slug}}/README.rst [hunk 1]
```diff
@@ -5,29 +5,12 @@
...
-.. code:: console
-
-    $ createdb {{ cookiecutter.project_slug }}_django
-
-Run migrations:
-
-.. code:: console
-
-    $ python manage.py migrate
+    $ blanc-clone {{ cookiecutter.project_slug }}
```
Comment (zatan) [inline {{cookiecutter.project_slug}}/README.rst]: Is it worth adding how to get blanc-clone?

**Comment by zatan** [comment]
Can we format MakeFile for better readability? 
At the moment we are mixing 4,3,2 blank spaces between entries.

**Comment by takeontom** [comment]
@zatan I can't see any inconsistencies?

3 blank lines between a section (e.g. Common targets for Blanc projects)
2 between groups (e.g. Virtual Environments)
1 between individual tasks (e.g. venv-check)

**Comment by zatan** [comment]
Need a new pair of :eyes:

**Comment by takeontom** [comment]
Going to rename the blanc tools before merging this in, as it refers to them and want to do it with the minimum of PRs.

**Pull request merged**